packages/cgb-scripts/template/src/init.php

```diff
@@ -24,24 +24,24 @@
  */
 function <% blockNamePHPLower %>_cgb_block_assets() {
 	// Styles.
-	wp_enqueue_style(
+	wp_register_style(
 		'<% blockNamePHPLower %>-cgb-style-css', // Handle.
 		plugins_url( 'dist/blocks.style.build.css', dirname( __FILE__ ) ), // Block style CSS.
 		array( 'wp-blocks' ) // Dependency to include the CSS after it.
 		// filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.style.build.css' ) // Version: filemtime — Gets file modification time.
 	);
 
 	// Block scripts.
-	wp_enqueue_script(
+	wp_register_script(
 		'<% blockNamePHPLower %>-cgb-block-js', // Handle.
 		plugins_url( '/dist/blocks.build.js', dirname( __FILE__ ) ), // Block.build.js: We register the block here. Built with Webpack.
-		array( 'wp-blocks', 'wp-i18n', 'wp-element' ) // Dependencies, defined above.
-		// filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.build.js' ), // Version: filemtime — Gets file modification time.
+		array( 'wp-blocks', 'wp-i18n', 'wp-element' ), // Dependencies, defined above.
+		false, // filemtime( plugin_dir_path( __DIR__ ) . 'dist/blocks.build.js' ), // Version: filemtime — Gets file modification time.
```

Comment:
Introduced `false` as the 4th parameter because the next parameter – `true // Enqueue the script in the footer.` – was useless because it was being taken as the 4th because of the comment.

Reply:
That is not useless. Read this https://github.com/ahmadawais/create-guten-block/pull/43

Reply:
The way I see it, 4th parameter never made it to the master branch. See this merge commit of the PR: https://github.com/ahmadawais/create-guten-block/commit/d506e27b306206ca0b95c434aeb6c3c44c9c86a5#diff-4317d120593768bbdedb26f6949dadad